xmodule/html_block.py

```diff
@@ -1,7 +1,6 @@
 # lint-amnesty, pylint: disable=missing-module-docstring
 
 import copy
-import json
```

Comment:
This was unused? Gross. Probably shouldn't be in this PR though

Reply:
It was causing linting errors since the file was changed, so I need it.

Reply:
I don't see anything changed in this file except for this line? Am I missing something?